programmers2/실패율.js

```diff
@@ -0,0 +1,13 @@
+function solution(N, stages) {
+    const arr = Array.from(Array(N), () => new Array(2));
+    let allPlayer = stages.length;
+    for (let i = 1; i < N + 1; i++) {
```

Comment:
```suggestion
    for (let i = 1; i < N + 1; i += 1) {
```
[no-plusplus 규칙](https://eslint.org/docs/rules/no-plusplus)